cli/cmd/install.go

```diff
@@ -397,7 +397,7 @@ func render(w io.Writer, values *l5dcharts.Values, stage string, options valuesp
 	}
 
 	if stage == "" || stage == controlPlaneStage {
-		overrides, err := renderOverrides(values, values.Namespace, false)
```

Comment:
I think the marshal and unmarshal of rawValues above can be replaced with `ToMap()`

Reply:
Ahhh, Right. Updated